### ISSUE TITLE
Feature Request: Configurable width for .pcr-app

### ISSUE THREAD

**Issue by elgordino** [Simonwep/pickr#88]
#### Do you want to request a **feature** or report a **bug**? 

Feature

#### What is the current behavior?

The width of the pickr is fixed at 28.5rem. When displaying only swatches it is rather wide. I would like the option to be able to customise the width.

Currently like this:

![Screenshot 2019-05-12 at 12 11 44](https://user-images.githubusercontent.com/19666042/57581426-304d9380-74af-11e9-9c9e-33a0f242c680.png)

#### What is the expected behavior?

I would like to be able to set a narrower width like this:

![Screenshot 2019-05-12 at 12 12 50](https://user-images.githubusercontent.com/19666042/57581441-48bdae00-74af-11e9-8ac5-69ab3bd171d1.png)

**Comment by Simonwep** [comment]
You can change the width of `.pcr-app` any time using custom css like
```css
.pcr-app {
    width: 15em;
}
```

This would change the width of all pickr's on the page. Specifying the width of a single one would require a extra class which could be an additional option...

Btw. The spaces between the swatches on the first Screenshot are too thin, which browser did you use? It uses flex as fallback, but there should be always some spacing like in your second screenshot :suspect:

**Comment by elgordino** [comment]
Hey Simon. I would like, if possible, to be able to set it on a per pickr basis. Would it be possible to add an option to add an extra class name to the created element?

Both the first and second screenshots are from Chrome 73 on Mac Mojave, the only difference being I changed the width on .pcr-app.

**Comment by Simonwep** [comment]
Splendid :)

**Comment by Simonwep** [comment]
Wups, will be added!

**Comment by elgordino** [comment]
Confused me for a moment there ;-)

Thanks @Simonwep!

**Comment by elgordino** [comment]
Setting  `grid-template-columns: repeat(auto-fit,2.1rem);` on pcr-swatches seem to look better?

**Comment by Simonwep** [comment]
Ugh, yeah accidentally  pressed close-and-comment. GitHub should request a user-confirmation for this haha
Okay, a `appClass` option got added in 7a16947120cbe686a0f552a7324ff37469682797:
```js
{
    appClass: 'my-custom-class'
}
```

Will be simply applied to the `pcr-app` element.
Regarding the grid, the width's between the swatches would be not the same as on the y-axis:
![image](https://user-images.githubusercontent.com/30767528/57581947-45322300-74bf-11e9-9d62-8f1f6d38d973.png)
I have no clue why there's no gap :suspect: The swatches itself have a `margin` of ~`2px`. So technically this shouldn't be possible...

**Comment by elgordino** [comment]
Thanks for the change!

That's weird. The horizontal and vertical separation looks good for me with `repeat(auto-fit,2.1rem)` with both the regular and narrower width:

![Screenshot 2019-05-12 at 13 22 53](https://user-images.githubusercontent.com/19666042/57582151-2335a200-74b9-11e9-8ff9-5de13e70ad44.png)


![Screenshot 2019-05-12 at 13 23 03](https://user-images.githubusercontent.com/19666042/57582157-29c41980-74b9-11e9-9c62-4bd581532b63.png)

I'm wondering if I've got something else in my CSS somewhere that's causing this... I have zero experience of flex-grid, we're still supporting IE11 (although not for this bit of the app!).

**Comment by elgordino** [comment]
Thanks @Simonwep !

**Comment by Simonwep** [comment]
@elgordino Just published [0.5.1](https://www.npmjs.com/package/@simonwep/pickr/v/0.5.1) where `appClass` is available :)

**Comment by elgordino** [comment]
Awesome! Thanks!
